README.md

```diff
@@ -154,6 +203,57 @@ We enthusiastically welcome contributions and feedback. You should clone the rep
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+### Clone
+
+MSAL uses a submodule for a "common" library shared by MSAL, ADAL and the Microsoft authentication library
+
+```cmd
+git clone https://github.com/AzureAD/microsoft-authentication-library-for-android.git msal
+cd msal
+git submodule init
+git submodule update
+```
+
+### MSAL Flavors
+
+```gradle
+ flavorDimensions "main"
+
+productFlavors {
+    // The 'local' productFlavor sources common from mavenLocal and is intended to be used
+    // during development.
+    local {
+        dimension "main"
+        versionNameSuffix "-local"
+    }
+
+    // Intended for nightly builds
+    snapshot {
+        dimension "main"
+    }
+
+    // The 'dist' productFlavor sources common from a central repository and is intended
+    // to be used for releases.
+    dist {
+        dimension "main"
+    }
+}
+```
+
+### Build
+
+```cmd
+gradlew assembleLocalDebug
```

Comment:
I think this has to be ./gradlew

Reply:
+1 We should favor the project wrapper instead of whatever's on the PATH